src/events/guildMemberAdd.ts

```diff
@@ -0,0 +1,15 @@
+import { GuildMember, TextChannel } from 'discord.js';
+
+import { WELCOME } from '../constants';
+
+export const guildMemberAddEvent = async (member: GuildMember) => {
+	const formattedMessage = WELCOME.welcomeMessage
+		.replace(/\{userMention\}/g, `<@${member.id}>`) // @name
+		.replace(/\{userTag\}/g, member.user.tag); // name#1234
+
+	const channel = (await member.guild.channels.get(
+		process.env.WELCOME_CHANNEL,
+	)) as TextChannel;
```

Comment:
👎 no as allowed

Reply:
Well, if D.JS fixed their typing yet then you can just check channel.type and it will figure it out

Reply:
Yep, but tbh its not too bad in this instance.